zebedee-cms/src/main/java/com/github/onsdigital/zebedee/session/store/exceptions/SessionsDecodeException.java

```diff
@@ -0,0 +1,10 @@
+package com.github.onsdigital.exceptions;
+
+public class SessionsDecodeException extends RuntimeException{
+    public SessionsDecodeException(String message) {
+        this(message, null);
+    }
+    public SessionsDecodeException(String message, Throwable cause) {
+        super(message, cause);
+    }
+}
```

Comment:
Update this so it extends `SessionStoreException` instead of `Exception` - see previous comment.